README.md

```diff
@@ -3,3 +3,31 @@ Scripts for indexing data into ODC instances
 
 # GDAL3
 Please note - we are aware that using these scripts with GDAL 3.0+ can cause Latitudes and Longitudes to become transposed. When using these scripts, please use a python environment with an earlier version of GDAL.
+
+
+# odc-product-delete (In beta)
+SQL scripts to delete ODC products and related records in ODC, Explorer and OWS DB. Each script takes product_name as an input parameter.
+
+Usage:
```

Comment:
Let's make it explicit how you actually run the three scripts together. How is it expected to be used?